cleanslate/doma/tasks.py

```diff
@@ -8,7 +8,7 @@
 from .models import Event
 
 account_sid = "AC3c7a087cae83e671986819994e5cd5c9"
-auth_token = "269e1c35ee3f9778732275985fc6c1b5"
```

Comment:
I also requested a new token from Twilio, set that as the primary, and deleted this one associated with my account. Embarrassed. oh well.